core/frontend/src/tile/map/MapLayerSources.ts

```diff
@@ -25,16 +25,37 @@ export enum MapLayerSourceStatus {
   RequireAuth,
 }
 
+/** JSON representation of a map layer source.
+ * @internal
+ */
+interface MapLayerSourceProps extends MapLayerProps
```

Comment:
Weird that this inherits properties like `visible` that it are not used by MapLayerSource? Is there really an inheritance relationship between them or just a coincidental similarity?